src/debug.ts

```diff
@@ -106,16 +104,40 @@ async function createTestConfigurations(ctx: FolderContext): Promise<vscode.Debu
             cwd: `\${workspaceFolder:${ctx.folder.name}}`,
             preLaunchTask: `swift: Build All`
         }];
+    } else if (process.platform === 'win32') {
+        // On Windows, add XCTest.dll to the PATH,
+        // and then run the .xctest bundle from the .build directory.
+        let xctestPath: string;
+        try {
+            xctestPath = await getXCTestDLLPath();
+        } catch (error) {
+            vscode.window.showErrorMessage(
+                `I was unable to create a debug configuration for testing ` + 
+                `because I cannot find XCTest.dll. ${error}` 
+            );
+            return [];
+        }
+        return [{
+            type: "lldb",
```

Comment:
Are we always generating dwarf debug information?  Are you also passing `-Xlinker -debug:dwarf` when building?  Without that, this is not particularly useful.

Reply:
@svanimpe could this be why your windows debug sessions aren't working

Reply:
Yes, that fixes part of it. If I add those build flags, I can set breakpoints. However, I still can't resolve any variables (https://bugs.swift.org/browse/SR-14243)

Reply:
Is that with CoreLLDB setup for swift?

Reply:
Yes, I get that error when I set `liblldb.dll`. I also get the same error when running the `LLDB: Use Alternate Backend` command.

With an empty `lldb.library` setting, and these build flags, I can set breakpoints, but not inspect any variables.